pt/lessons/basics/enum.md

```diff
@@ -9,13 +9,33 @@ Um conjunto de algoritmos para fazer enumeração em coleções.
 
 ## Enum
 
-O módulo `Enum` inclui mais de cem funções para trabalhar com as coleções que aprendemos na última lição.
+O módulo `Enum` inclui mais de 70 funções para trabalhar com enumeráveis. Todas as coleções que aprendemos na [lição anterior](../collections/), com exceção das tuplas, são enumeráveis.
 
-Essa lição somente mostrará um subconjunto das funções disponíveis, para ver a lista completa de funções visite a documentação oficial [`Enum`](https://hexdocs.pm/elixir/Enum.html); para enumeração preguiçosa use o módulo[`Stream`](https://hexdocs.pm/elixir/Stream.html).
+Esta lição abrangerá apenas um subconjunto das funções disponíveis, no entanto, podemos examiná-las pessoalmente.
+Vamos fazer um pequeno experimento no IEx.
+
+```elixir
+iex> Enum.__info__(:functions) |> Enum.each(fn({function, arity}) ->
+...>   IO.puts "#{function}/#{arity}"
+...> end)
+all?/1
+all?/2
+any?/1
+any?/2
+at/2
+at/3
+...
+```
+
+Usando isso, é claro que temos uma grande quantidade de funcionalidades, e isso é por uma razão.
+A enumeração é o núcleo da programação funcional e é uma coisa incrivelmente útil.
+Ao aproveitar isso combinado com outras vantagens de Elixir, como a documentação sendo um cidadão de primeira classe como acabamos de ver, também pode ser incrivelmente poderoso para o desenvolvedor.
+
+Para obter uma lista completa de funções, visite a documentação oficial do [`Enum`](https://hexdocs.pm/elixir/Enum.html); Para enumeração preguiçosada use o módulo [`Stream`](https://hexdocs.pm/elixir/Stream.html).
```

Comment:
I believe the word "Para" could be changed to lowercase in this case - after the semicolon. What do you think?
Also, there is a small typo in "preguiçosada" -> "preguiçosa".